### PR TITLE
IOSXE/show cts wireless profile policy {policy}

### PR DESCRIPTION
**Pull request by qduk**
## Description
Parser created for show cts wireless profile policy {policy}

## Motivation and Context
Takes unstructured data returned from command and puts it into structured data

## Screenshots:
Capture

## Checklist:
- [x] I have updated the changelog.
- [x] All new and existing tests passed.
- [x] All new code passed compilation.

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_cts.py [hunk 1]
```diff
@@ -1947,6 +1947,110 @@ def filter_lines(raw_output):
...
+        cts_ap_dict = {}
+        current_policy = ""
+
+        for line in output.splitlines():
+            line = line.strip()
+            m_profile_name = p_profile_name.match(line)
+            if m_profile_name:
+                # Policy Profile Name         		: xyz-policy
+                current_policy = m_profile_name.group("value")
+                cts_ap_dict.setdefault("policy_name", {} ).update({ current_policy: {} })
```
Comment (badalsoyantar) [inline src/genie/libs/parser/iosxe/show_cts.py:2030]: continue

**Review by badalsoyantar — APPROVED** [review]

**Review by badalsoyantar — CHANGES_REQUESTED** [review]
Needs schema review

File: src/genie/libs/parser/iosxe/tests/ShowCtsWirelessProfilePolicy/cli/equal/golden_output_expected.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+expected_output = {
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowCtsWirelessProfilePolicy/cli/equal/golden_output_expected.py:1]: schema looks good

**Review by tahigash — APPROVED** [review]

File: src/genie/libs/parser/iosxe/show_cts.py (hunk 1 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/show_cts.py:2030]: Updated

**Review by badalsoyantar — APPROVED** [review]

**Comment by tahigash** [comment]
Thank you for your contribution!

**Pull request merged**